gradle.properties

```diff
@@ -11,6 +11,7 @@
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=2g
```

Comment:
More on that: https://github.com/Kotlin/dokka/issues/1405#issuecomment-1443665006